src/main/java/org/team1515/botmitzvah/Commands/Autonomous/AutoCommand.java

```diff
@@ -1,5 +1,29 @@
 package org.team1515.botmitzvah.Commands.Autonomous;
 
-public class AutoCommand {
-    
-}
+import org.team1515.botmitzvah.RobotContainer;
+import org.team1515.botmitzvah.Robot;
+import org.team1515.botmitzvah.Commands.AutoAlign;
+import org.team1515.botmitzvah.Commands.DriveDist;
+import org.team1515.botmitzvah.Subsystems.Drivetrain;
+//add other subsystems
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+
+public class AutoCommand extends SequentialCommandGroup {
```

Comment:
I would call the class something different than auto command, just because there may be multiple auto routines